[2022-08-11 19:47:21,618] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V6.Ingest_from_food_consumption_estimates manual__2022-08-11T19:47:15.827552+00:00 [queued]>
[2022-08-11 19:47:21,645] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V6.Ingest_from_food_consumption_estimates manual__2022-08-11T19:47:15.827552+00:00 [queued]>
[2022-08-11 19:47:21,646] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 19:47:21,647] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-11 19:47:21,648] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 19:47:21,666] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Ingest_from_food_consumption_estimates> on 2022-08-11 19:47:15.827552+00:00
[2022-08-11 19:47:21,676] {standard_task_runner.py:52} INFO - Started process 3847 to run task
[2022-08-11 19:47:21,691] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG_V6', 'Ingest_from_food_consumption_estimates', 'manual__2022-08-11T19:47:15.827552+00:00', '--job-id', '546', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpgh1c_wg2', '--error-file', '/tmp/tmpy7mab9uh']
[2022-08-11 19:47:21,694] {standard_task_runner.py:80} INFO - Job 546: Subtask Ingest_from_food_consumption_estimates
[2022-08-11 19:47:21,835] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG_V6.Ingest_from_food_consumption_estimates manual__2022-08-11T19:47:15.827552+00:00 [running]> on host ce1be8c59d56
[2022-08-11 19:47:21,939] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG_V6
AIRFLOW_CTX_TASK_ID=Ingest_from_food_consumption_estimates
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T19:47:15.827552+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T19:47:15.827552+00:00
[2022-08-11 19:47:29,358] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table1.xlsx to s3-bucket-raw-usda
[2022-08-11 19:47:30,666] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table2.xlsx to s3-bucket-raw-usda
[2022-08-11 19:47:31,788] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table3.xlsx to s3-bucket-raw-usda
[2022-08-11 19:47:31,790] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 19:47:31,800] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG_V6, task_id=Ingest_from_food_consumption_estimates, execution_date=20220811T194715, start_date=20220811T194721, end_date=20220811T194731
[2022-08-11 19:47:31,834] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 19:47:31,860] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
